commit 6e91b9c16ff53fadf75a545aabe5907f1c046d2f
Author: Etsuro Fujita <efujita@postgresql.org>
Date:   Sat May 3 19:10:00 2025 +0900

    Fix typos in comments.
    
    Also adjust the phrasing in the comments.
    
    Author: Etsuro Fujita <etsuro.fujita@gmail.com>
    Author: Heikki Linnakangas <hlinnaka@iki.fi>
    Reviewed-by: Tender Wang <tndrwang@gmail.com>
    Reviewed-by: Gurjeet Singh <gurjeet@singh.im>
    Reviewed-by: Michael Paquier <michael@paquier.xyz>
    Discussion: https://postgr.es/m/CAPmGK17%3DPHSDZ%2B0G6jcj12buyyE1bQQc3sbp1Wxri7tODT-SDw%40mail.gmail.com
    Backpatch-through: 15

diff --git a/src/backend/utils/activity/pgstat_database.c b/src/backend/utils/activity/pgstat_database.c
index 52d82d25352..b31f20d41bc 100644
--- a/src/backend/utils/activity/pgstat_database.c
+++ b/src/backend/utils/activity/pgstat_database.c
@@ -419,12 +419,12 @@ pgstat_reset_database_timestamp(Oid dboid, TimestampTz ts)
 }
 
 /*
  * Flush out pending stats for the entry
  *
- * If nowait is true, this function returns false if lock could not
- * immediately acquired, otherwise true is returned.
+ * If nowait is true and the lock could not be immediately acquired, returns
+ * false without flushing the entry.  Otherwise returns true.
  */
 bool
 pgstat_database_flush_cb(PgStat_EntryRef *entry_ref, bool nowait)
 {
 	PgStatShared_Database *sharedent;
diff --git a/src/backend/utils/activity/pgstat_function.c b/src/backend/utils/activity/pgstat_function.c
index bc4e254d02d..6214f93d36e 100644
--- a/src/backend/utils/activity/pgstat_function.c
+++ b/src/backend/utils/activity/pgstat_function.c
@@ -184,12 +184,12 @@ pgstat_end_function_usage(PgStat_FunctionCallUsage *fcu, bool finalize)
 }
 
 /*
  * Flush out pending stats for the entry
  *
- * If nowait is true, this function returns false if lock could not
- * immediately acquired, otherwise true is returned.
+ * If nowait is true and the lock could not be immediately acquired, returns
+ * false without flushing the entry.  Otherwise returns true.
  */
 bool
 pgstat_function_flush_cb(PgStat_EntryRef *entry_ref, bool nowait)
 {
 	PgStat_FunctionCounts *localent;
diff --git a/src/backend/utils/activity/pgstat_relation.c b/src/backend/utils/activity/pgstat_relation.c
index eeb2d43cb10..28587e2916b 100644
--- a/src/backend/utils/activity/pgstat_relation.c
+++ b/src/backend/utils/activity/pgstat_relation.c
@@ -804,12 +804,12 @@ pgstat_twophase_postabort(TransactionId xid, uint16 info,
 }
 
 /*
  * Flush out pending stats for the entry
  *
- * If nowait is true, this function returns false if lock could not
- * immediately acquired, otherwise true is returned.
+ * If nowait is true and the lock could not be immediately acquired, returns
+ * false without flushing the entry.  Otherwise returns true.
  *
  * Some of the stats are copied to the corresponding pending database stats
  * entry when successfully flushing.
  */
 bool
diff --git a/src/backend/utils/activity/pgstat_subscription.c b/src/backend/utils/activity/pgstat_subscription.c
index 830d408c61a..f9a1c831a07 100644
--- a/src/backend/utils/activity/pgstat_subscription.c
+++ b/src/backend/utils/activity/pgstat_subscription.c
@@ -94,12 +94,12 @@ pgstat_fetch_stat_subscription(Oid subid)
 }
 
 /*
  * Flush out pending stats for the entry
  *
- * If nowait is true, this function returns false if lock could not
- * immediately acquired, otherwise true is returned.
+ * If nowait is true and the lock could not be immediately acquired, returns
+ * false without flushing the entry.  Otherwise returns true.
  */
 bool
 pgstat_subscription_flush_cb(PgStat_EntryRef *entry_ref, bool nowait)
 {
 	PgStat_BackendSubEntry *localent;
